rca/static_src/sass/layout/_header.scss

```diff
@@ -26,7 +26,80 @@
     }
 
     &__menu-toggle {
-        margin-right: ($gutter);
+        margin-top: $grid / 4;
```

Comment:
I've previously been told to wrap all calculations in () for easier reading, so $grid / 4 would become ($grid / 4), don't know if you want to follow this convention? Would make sense for us to be consistent. 

I'm happy either way, just as long as are both doing the same.

Reply:
can do 👍